spec/integration/create_subscriber_list_spec.rb

```diff
@@ -101,22 +101,22 @@
     end
 
     it "returns an error if tag isn't an array" do
-      create_subscriber_list(
+      create_and_joined_facet_subscriber_list(
         tags: { topics: { any: "oil-and-gas/licensing" } },
       )
 
       expect(response.status).to eq(422)
     end
 
     it "successfully creates two SubscriberList objects with the same title" do
-      create_subscriber_list(title: "oil and gas", links: { taxons: { any: ["oil-and-gas"] } })
-      create_subscriber_list(title: "oil and gas", links: { policies: { any: ["oil-and-gas/licensing"] } })
+      create_and_joined_facet_subscriber_list(title: "oil and gas", links: { taxons: { any: ["oil-and-gas"] } })
+      create_and_joined_facet_subscriber_list(title: "oil and gas", links: { policies: { any: ["oil-and-gas/licensing"] } })
```

Comment:
I'm a little nervous even suggesting it, but... would the method name be clearer if we renamed it to: `create_AND_joined_facet_subscriber_list`?

I know that goes against the standard rules of the language but it currently reads as "create and joined", which is kind of like "created and joined", and takes a bit of rethinking to actually understand what it's doing. Whereas with the capitals it hopefully reads as "create an AND-joined list" a bit more clearly.

Feel free to burn this idea with fire.